.circleci/config.yml

```diff
@@ -23,6 +23,9 @@ common: &common
         requires:
           - python/pip-run-tests
 
+    - python/typing:
+        packages: sybil tests
```

Comment:
The CI results show errors on `tests`. I believe you don't want `mypy` running against `tests`, and I think that means this line should therefore be `packages: sybil`.

Reply:
Ah yes, that's right! Can you just remove the `tests` when you next rebase this branch on master>
I'd forgotten we'd agreed that, yay! we can lose all the eye-bleed ` -> None` on every test 🎉